README.md

```diff
@@ -1,2 +1,16 @@
 # Aleph-Testnet
-Aleph Zero Blockchain Helm chart with all the necessary tooling to run it locally.
+
+Aleph Zero Blockchain python scripts for running TestNet and DevNet.
+
+# Usage instructions
+
+- Create an account on AWS, set up credentials, and a default region as described [here](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html#configuration).
+- Put keys for ssh in key_pairs.
+- Install packages needed for orchestrating experiments: GNU parallel, zip, unzip and Python 3 packages: fabric, boto3, ipython, tqdm, matplotlib.
+- Then, `run ipython -i shell.py`. This opens a shell with procedures orchestrating experiments.
+  The main procedures are `run_protocol(n_processes, regions, instance_type)` and
+  `run_devnet(n_processes, regions, instance_type)` that run `n_processes` spread
```

Comment:
A minimal working example would be nice. I think it's `run_devnet(4)`, but I'm not sure.

Reply:
It would also be nice to know how to turn this off. <____<